catkit2/services/thorlabs_cubes/thorlabs_cubes.py

```diff
@@ -0,0 +1,108 @@
+from catkit2.testbed.service import Service
+
+import thorlabs_apt as apt
+import numpy as np
+
+
+class ThorlabsCubes(Service):
+
+    def __init__(self):
+        super().__init__('thorlabs_cubes')
+
+        self.cube_serial_number = self.config['serial_number']
+
+        self.command = self.make_data_stream('command', 'float64', [1], 20)
+        self.current_position = self.make_data_stream('current_position', 'float64', [1], 20)
+
+        # Initialization of the parameters
+        self.motor = None
+        self.cube_model = None
+        self.motor_unit = None
+        self.motor_max = None
+        self.motor_min = None
+
+    def open(self):
+        """
+            Open the motor and cube.
+
+            This will also check if the configured characteristics are matching the one the cube have.
+        """
+        self.motor = apt.Motor(self.cube_serial_number)
+
+        # Retrieve min, max, unit and model of the motor and cube

```

Comment:
Since you want to do a comparison between these parameters as they come from the device itself with what we feed it from the configfile, be more explicit in this comment, e.g.: "Retrieve the min, max, unit and model from the connected device."